src/utils/content.ts

```diff
@@ -30,7 +30,22 @@ export class ContentWrapper {
     })
   }
 
-  get = async (entity: string) => {
+  // get<T> = async (entity: string) : Promise<T> => {
+  //   const client = this.client
+
+  //   const entries = await client.getEntries({
+  //     content_type: entity,
+  //   })
+  //   return entries
+  // }
+
+  async get<T>(entity: string): Promise<T> {
+    const client = this.client
+    const entries = await client.getEntries({ content_type: entity })
+    return entries as unknown as T
```

Comment:
```suggestion
    return entries.items.map((item) => item.fields) as unknown as T
```
This will fix a lot of weird typing issues and make things a lot cleaner in the `getCompany` procedure^^